src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/KeyStatisticServiceTests.cs

```diff
@@ -346,6 +346,164 @@ await using (var context = InMemoryContentDbContext(contextId))
+    {
+        var release = new Release();
+
+        var contextId = Guid.NewGuid().ToString();
+        await using (var context = InMemoryContentDbContext(contextId))
+        {
+            await context.Releases.AddRangeAsync(release);
+            await context.SaveChangesAsync();
+        }
+
+        await using (var context = InMemoryContentDbContext(contextId))
+        {
+            var keyStatisticService = SetupKeyStatisticService(context);
+            var result = await keyStatisticService.CreateKeyStatisticText(
+                Guid.NewGuid(),
+                new KeyStatisticTextCreateRequest
+                {
+                    Title = "title",
+                    Statistic = "Over 9000!",
+                });
+
+            result.AssertNotFound();
+        }
+    }
+
+    [Fact]
+    public async Task CreateKeyStatisticText_Order()
+    {
+        var release = new Release
+        {
+            KeyStatistics = new List<KeyStatistic>
+            {
+                new KeyStatisticText { Order = 0 },
+                new KeyStatisticText { Order = 1 },
+                new KeyStatisticDataBlock { Order = 2 },
+            },
+        };
+
+        var contextId = Guid.NewGuid().ToString();
+        await using (var context = InMemoryContentDbContext(contextId))
+        {
+            await context.Releases.AddRangeAsync(release);
+            await context.SaveChangesAsync();
+        }
+
+        await using (var context = InMemoryContentDbContext(contextId))
+        {
+            var keyStatisticService = SetupKeyStatisticService(context);
+
+            var result = await keyStatisticService.CreateKeyStatisticText(
+                release.Id,
+                new KeyStatisticTextCreateRequest
+                {
+                    Title = "title",
+                    Statistic = "Over 9000!",
+                });
+
+            var viewModel = result.AssertRight();
+
+
+            Assert.Equal("title", viewModel.Title);
+            Assert.Equal("Over 9000!", viewModel.Statistic);
+            Assert.Null(viewModel.Trend);
+            Assert.Null(viewModel.GuidanceTitle);
+            Assert.Null(viewModel.GuidanceText);
+            Assert.Equal(3, viewModel.Order);
+            Assert.InRange(DateTime.UtcNow.Subtract(viewModel.Created).Milliseconds, 0, 1500);
+            Assert.Null(viewModel.Updated);
+        }
+
+        await using (var context = InMemoryContentDbContext(contextId))
+        {
+            var keyStatistics = context.KeyStatistics
+                .OrderBy(ks => ks.Order)
+                .ToList();
+            Assert.Equal(4, keyStatistics.Count);
+
+            Assert.Equal(0, keyStatistics[0].Order);
+            Assert.Equal(1, keyStatistics[1].Order);
+            Assert.Equal(2, keyStatistics[2].Order);
+
+            var keyStatText = Assert.IsType<KeyStatisticText>(keyStatistics[3]);
+
+            Assert.Equal("title", keyStatText.Title);
```

Comment:
As above- probably only need to be checking its Order here (as well as checking it's the right KeyStat!).